src/classes/query-builder.js

```diff
@@ -1,89 +1,136 @@
+const capitalise = word => {
+  return word.charAt(0).toUpperCase() + word.slice(1)
+}
+
 module.exports = class QueryFilter {
-  constructor (knex) {
-    this.knex = knex
+  constructor (queryBuilder, Model) {
+    this.Model = Model
+    this.queryBuilder = queryBuilder
+    this.depth = 0 // nested query depth
+  }
+
+  _orBuilder (array) {
```

Comment:
```js
/**
 * @private
 **/
orBuilder (array) {}
```

Basically how ember does it - it really mostly affects the auto-API documentation, or if you bother to read the comments when reading through the code.